coopr-docs/docs/source/rest/metrics.rst

```diff
@@ -81,3 +83,149 @@ Example
+   :header-rows: 1
+
+   * - Status Code
+     - Description
+   * - 200 (OK)
+     - If update was successful
+   * - 403 (FORBIDDEN)
+     - If the user is forbidden from getting node usage metrics.
+
+
+
+
+API Endpoints and Examples
+==========================
+
+
+General Usage
+-------------
+
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:admin'
+        -H 'Coopr-ApiKey:<apikey>'
+        -H 'Coopr-TenantID:superadmin'
+        http://<server>:<port>/<version>/metrics/nodes/usage
+
+ $ {"start":1424475097,"end":1424485733,"data":[{"time":1424475097,"value":29035}]}
+
+
+The value at the end of the resulting JSON is the number of node seconds for all nodes combined.
+
+As an example, if you have a 3 node cluster and a 1 node cluster (total 4 nodes), every 60 seconds,
+you will have 240 (4 nodes x 60 seconds) additional node seconds.  The following two lines are examples
+of the JSON results returned when running a general API node usage call every 60 seconds:
+
+.. code-block:: bash
+
+ {"start":1424475097,"end":1424486443,"data":[{"time":1424475097,"value":31872}]}
+ {"start":1424475097,"end":1424486503,"data":[{"time":1424475097,"value":32112}]}
+
+
+
+Time Ranges
+-----------
+
+Start and end values are expressed in Unix time (a.k.a. seconds since epoch, or the beginning of Unix time:
+00:00:00 Coordinated Universal Time (UTC), Thursday, 1 January 1970).
+Unix time may be checked on most Unix systems by typing ``date +%s`` on the command line.
+
+With the general usage example, the JSON response to general node hour usage shows the start and end values returned. 
+If the start is not specified, the start time is the exact date and time when the very first 
+task (ever) completes. The end date, if not specified, is the current date and time.
+
+
+Start Time Only
+^^^^^^^^^^^^^^^
+
+To obtain node usage hours since a specific date and time, we would add ``start=x`` where x is a Unix time.
+
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:admin'
+        -H 'Coopr-ApiKey:<apikey>'
+        -H 'Coopr-TenantID:superadmin'
+        http://<server>:<port>/<version>/metrics/nodes/usage?start=1424393580
+ 
+ $ {"start":1424393580,"end":1424485774,"data":[{"time":1424393580,"value":29199}]}
+
+
+End Time Only
+^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:admin'
+        -H 'Coopr-ApiKey:<apikey>'
+        -H 'Coopr-TenantID:superadmin'
+        http://<server>:<port>/<version>/metrics/nodes/usage?end=1424482780
+ 
+ $ {"start":1424475097,"end":1424482780,"data":[{"time":1424475097,"value":17220}]}
+
+
+Time Range (Start and End Time)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:admin'
+        -H 'Coopr-ApiKey:<apikey>'
+        -H 'Coopr-TenantID:superadmin'
+        http://<server>:<port>/<version>/metrics/nodes/usage?start=1424481097&end=1424482780
+ 
+ $ {"start":1424481097,"end":1424486196,"data":[{"time":1424481097,"value":20396}]}
+
+
+
+Specifying a User
+-----------------
```

Comment:
So... the answer is **yes** and **no** (as usual)... The superadmin gets an aggregate (select * from table) whereas the individual tenants can only see themselves (select * from table where tenant_id=me)....
```
$ for i in superadmin cask-dev cask-ops ; do echo $i ; curl -H coopr-userid:admin -H coopr-tenantid:$i http://coopr-dev:55054/v2/metrics/nodes/usage?user=admin ; echo ; done
superadmin
{"start":1410815909,"end":1425394885,"data":[{"time":1410815909,"value":181607421}]}
cask-dev
{"start":1410984410,"end":1425394885,"data":[{"time":1410984410,"value":177621964}]}
cask-ops
{"start":1422066998,"end":1422371161,"data":[{"time":1422066998,"value":820843}]}
```